locales/en-US/editor.properties

```diff
@@ -101,6 +102,10 @@ SAVE_AND_OVERWRITE=Overwrite
 DELETE=Delete
 BUTTON_YES=Yes
 BUTTON_NO=No
+# "Use Imported" as in "Use the imported version of this file vs. the existing one"
+USE_IMPORTED=Use Imported
+# "Use Existing" as in "Use the existing version of this file vs. the imported one"
```

Comment:
nit: "Keep Existing" vs. "Use Existing" and "Keep the existing" vs. "Use the existing"